### DISCUSSION TITLE
What are the names of Cinderella’s stepsisters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#739] [Galaxy Trivia] [answered]
**Bonus:** Which Disney Princess talks to the most animals?

**Comment by SANKARAMDAS** [comment]
Stepsisters Anastasia and Drizella Tremaine.
Cinderella and it is perhaps through her gentle nature working in combination with her appreciation and regard for all living creatures that have fostered her endearing ability to actually communicate with creatures of all kinds.

**Comment by peterbaccaro** [comment, marked as answer]
In the classic fairy tale of Cinderella, the names of her stepsisters are **Anastasia** and **Drizella**.
Cinderella's stepsisters are typically not given surnames in the original fairy tale or in most adaptations. However, in some adaptations, they have been given the surnames Tremaine or Gothel. In the Disney animated movie version, their full names are Anastasia Tremaine and Drizella Tremaine.

**Snow White** is the Disney Princess who is known for talking to animals the most. In the classic 1937 Disney animated film "Snow White and the Seven Dwarfs," Snow White is famously shown singing and speaking to various forest animals, including birds, rabbits, deer, and squirrels. This ability to communicate with animals is one of her defining traits as a character. Other Disney Princesses, such as Cinderella and Aurora, also have a connection with animals, but Snow White is most associated with this characteristic.
